tests/functional/firefox/test_whatsnew.py

```diff
@@ -29,10 +29,10 @@ def test_send_to_device_fails_when_missing_required_fields(base_url, selenium):
 
 @pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
 @pytest.mark.nondestructive
-def test_firefox_rocket_qr_code(base_url, selenium):
+def test_firefox_rocket_send_yourself(base_url, selenium):
     page = FirefoxWhatsNewPage(selenium, base_url, locale='id').open()
-    assert not page.send_to_device.is_displayed
-    assert page.is_qr_code_displayed
+    assert not page.is_qr_code_displayed
+    assert page.send_yourself.is_displayed
```

Comment:
As this page no longer features a QR code, I think we should change the test to test the functionality of the form, rather than just assert that it is displayed. You can see an example of this in the [existing whatsnew page test](https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/test_whatsnew.py#L16-L19). You should just need to change `sent_to_device` to `send_yourself`.